kalite/settings/base.py

```diff
@@ -188,14 +188,12 @@
 # the user running kalite and should be in a user-data
 # storage place.
 
-USER_WRITABLE_LOCALE_DIR = os.path.join(USER_DATA_ROOT, 'locale')
-KALITE_APP_LOCALE_DIR = os.path.join(USER_DATA_ROOT, 'locale')
+USER_WRITABLE_LOCALE_DIR = os.path.join(USER_DATA_ROOT, 'project', 'locale')
```

Comment:
We are no longer serving `django.mo` etc files from content packs. Content packs will only contain subtitles.

Meanwhile, another important aspect is that we cannot and should not change location folders of existing unpacked subtitles, because that will mean that existing content packs have to be re-installed - that cannot happen in an upgrade.

This PR should only remove everything that has to do with translation files (`.mo`) from content packs and instead use the location of `kalite/project/locale`, which is a non-existing location until you put something there :)

I think you have confused it with `USER_DATA_ROOT`, which is to be understood as `~/.kalite`

Reply:
I think `USER_WRITABLE_LOCALE_DIR` and `KALITE_APP_LOCALE_DIR` are entirely useless paths and you should factor them out completely.

The definition of `get_subtitle_path` doesn't seem to rely on these at all, `.kalite/locale` was only for translation files IIRC.